app/backend/models/comment.js

```diff
@@ -0,0 +1,28 @@
+const {mongoose} =  require('../db');  // The mongodb connector library
+
+let Comment = mongoose.model('Comment', {
+
+  userId: {
+    type: String,
+    require: true
+  },
+
+  related: {
```

Comment:
We may need to discuss how to establish a relationship between a comment and the entity it's about. Only keeping the record of the entity the comment was made on seems inefficient, as that would make it necessary to check many collections for the given object id, which is redundant.